app.rb

```diff
@@ -0,0 +1,68 @@
+# app.rb
+require_relative 'person'
+require_relative 'book'
+require_relative 'rental'
+
+class LibraryApp
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'List of all books:'
+    @books.each { |book| puts "#{book.title} by #{book.author}" }
+  end
+
+  def list_all_people
+    puts 'List of all people:'
+    @people.each { |person| puts "#{person.correct_name} (ID: #{person.id}, Age: #{person.age})" }
+  end
+
+  def create_person(age, name = 'Unknown', parent_permission: true)
+    person = Person.new(age, name: name, parent_permission: parent_permission)
+    @people << person
+    puts "#{person.correct_name} has been created with ID #{person.id}."
+  end
```

Comment:
[REQUIRED] Kindly ensure that your app looks like the app displayed in the video Demo. When I attempt to create a person I don't get the first question which asks if I want to create a teacher or student. Hence I am forced to create a student. Kindly include this question and create ensure that your code allows a user to create a teacher or student.